.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "saturday"
```

Comment:
I think the Saturday runs are fine. It should make a single pr with many version updates right? not one per update

Reply:
I don't think so - it's likely that it will still make 1 PR for 1 update.

Reply:
Yep ... looks like one PR per explicit dependency https://github.com/dependabot/dependabot-core/issues/1190

Reply:
We have about 24 dependencies ... that shouldn't be too bad